config/routes.rb

```diff
@@ -164,6 +163,8 @@
          as: :create_verify_personal_key
     get '/account/two_factor_authentication' => 'accounts/two_factor_authentication#show'
 
+    get '/api/country-support' => 'country_support#index'
```

Comment:
I think `/:locale/api/` is a little confusing, I'd leave the method up top where it was before, and consider making locale an optional `GET` param ex `/api/country-support?locale=fr` and default to english

Since it has the same name, it would still get the same default translation behavior

Reply:
yea I was deciding between /locale/api or passing it in. but yea. will change it